README.md

```diff
@@ -80,12 +80,14 @@ The script `clean-old-results` can be used to get rid of undesired old results (
 
 ## How to contribute
 
-It is easy: just create a Pull Request with your contribution, and make sure that the unit tests are passing.
+It is easy: 
+- Make sure you have Git commit signing enabled. If you are not doing it already, check out the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
+- Make sure unit tests are passing
```

Comment:
```suggestion
- Make sure unit tests are passing.
```

![image](https://github.com/uyuni-project/terracumber/assets/12104291/292f356a-4eec-4644-8f5a-b8d187e195a2)

Reply:
Ah. This one one yes. Fixing it.